dfd-wrapper/src/main/java/com/revolut/rxdata/dfd_wrapper/DataFlowDelegate.kt

```diff
@@ -0,0 +1,109 @@
+package com.revolut.rxdata.dfd_wrapper
```

Comment:
Please call this class DataObservableDelegateWrapper, cause it wraps DOD not DFD. Also the module should probably be called dod-wrapper, not dfd wrapper.

Reply:
Done 😁